server/atob

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
Did you mean to add any of these files in server? I'm not sure how they ended up here.

Could you do a `git pull upstream master` on this branch locally and then push again? (they might have been removed)